libs/vm-modules/examples/crypto_price_prediction/script/bitcoin.etch

```diff
@@ -44,6 +44,15 @@ function build_graph(graph : Graph)
 
 endfunction
 
+function add_leading_zero_dim(tensor : Tensor)
```

Comment:
maybe add a comment explaining why this is needed - it's not immediately obvious